atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/persistentlock/LockStore.java

```diff
@@ -83,7 +79,9 @@ public void insertLockEntry(LockEntry lockEntry) {
     }
 
     private void backgroundCleanupTombstonedEntries(Set<LockEntry> tombstonedLockEntries) {
-        backgroundCleanExecutor.submit(() -> attemptToCleanupTombstonedEntries(tombstonedLockEntries));
+        Thread backgroundDeletionThread =
+                new Thread(() -> attemptToCleanupTombstonedEntries(tombstonedLockEntries), "Background deletions");
```

Comment:
Can you prefix thread name with AtlasDB so it's clear when mixed with application threads? Also, should this be daemon thread or not?